launcher/settingsView/csettingsview_moc.ui

```diff
@@ -558,31 +514,122 @@
      </item>
     </widget>
    </item>
-   <item row="11" column="6">
-    <widget class="QLabel" name="labelAutoSave">
+   <item row="8" column="7" colspan="2">
+    <widget class="QComboBox" name="comboBoxLanguage">
+     <item>
+      <property name="text">
+       <string>English</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>Deutsch (German)</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>Polska (Polish)</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>Русский (Russian)</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>Українська (Ukrainian)</string>
+      </property>
```

Comment:
I see that languages are met in several places. I suggest to add items into this combobox in constructor instead of having them statically

Reply:
This should be the only place with such form, as in - human-readable names of languages to select from.
Won't be possible to define languages in a single place though - mod schema, settings schema, translation files. And this is only what was added so far - there may be others in future (although I'll obviously would try to avoid duplication).

That being said, I do have plans for a second place to select language - some sort of a "welcome screen" for initial setup. So your suggestion makes sense...

Reply:
For now - keeping as it.
I agree that this is valid suggestion and will likely implement it a bit later, e.g. when working on "Welcome" window. Not yet sure where and how we'll need list this of languages, so keeping it as it until we do need it.
Mostly to avoid doing job twice if it turns out that I needed different approach here.